.vscode/launch.json

```diff
@@ -101,6 +101,31 @@
                     "path": "${workspaceFolder}/frontend"
                 }
             ]
+        },
```

Comment:
Did you mean to include these changes?

Reply:
Yessir, it's nice to be able to run Jest tests in VSCode!